core-plugins/docs/File-batchsource.md

```diff
@@ -20,6 +20,11 @@ If the format is 'blob', every input file will be read into a separate record.
 The 'blob' format also requires a schema that contains a field named 'body' of type 'bytes'.
 If the format is 'text', the schema must contain a field named 'body' of type 'string'.
 
+**Get Schema:** Auto-detects schema from file. Supported formats are: avro, parquet, csv, delimited,tsv.
```

Comment:
`Supported formats are: avro, parquet, csv, delimited,tsv.` - It also supports blob and text right?

Reply:
fixed